main.go

```diff
@@ -35,8 +33,6 @@ func main() {
 	}
 
 	rootCmd := root.New(cfg)
-	rootCmd.AddCommand(version.New(cfg))
```

Comment:
Any reason for this other than to keep main clean?

Reply:
Yeah - the doc generator needs the subcommands need to be added to pick them up. This moves the AddCommand's to the rootCmd creation to guarantee both main's have the same command tree.